### PR TITLE
Update roadmap for 2022

### PR DESCRIPTION
**Pull request by spowelljr**
Feel free to suggest changes and additions

### CODE REVIEW

File: site/content/en/docs/contrib/roadmap.en.md [hunk 1]
```diff
@@ -10,49 +10,25 @@ This roadmap is a living document outlining the major technical improvements whi
...
 
-- [x] Multi-node
-- [ ] IPv6
-- [ ] Usage documentation for 3 leading CNI providers
-- [ ] Automatically publish conformance test results after a release
+- [ ] Add Linux VM support
+- [ ] Add Mac M1 VM support
 
-## (#4) Cross-platform
+## (#4) Reduce Docker Desktop Depencency
```
Comment (afbjorklund) [inline site/content/en/docs/contrib/roadmap.en.md:30]: Possibly we could group "both" Docker items under one headline ?

1) remove the Docker Desktop requirement, on Mac and Win
2) continue supporting Docker as a container runtime (with CRI)

File: site/content/en/docs/contrib/roadmap.en.md [hunk 2]
```diff
@@ -10,49 +10,25 @@ This roadmap is a living document outlining the major technical improvements whi
...
-- [x] WSL2 support (no additional VM required)
-- [ ] Firecracker VM support
-- [x] Generic (SSH) driver support
+## (#5) libmachine Refactor
 
-## (#5) Reliable
-
-- [ ] Resource alerts
-- [ ] Time synchronization on HyperKit
 - [ ] Prototype post-libmachine implementation of minikube
```
Comment (afbjorklund) [inline site/content/en/docs/contrib/roadmap.en.md:34]: Will split this up further, we have had the "prototype" item shelved for years.

1. add new driver (with QEMU) to replace HyperKit , primarly for Mac arm64
2. fix the provisioner, remove legacy Swarm and add support for other runtimes

We need to move and cleanup the code as well, but that's not a visible feature.

File: site/content/en/docs/contrib/roadmap.en.md [hunk 3]
```diff
@@ -10,49 +10,25 @@ This roadmap is a living document outlining the major technical improvements whi
...
-# 2021
+# 2022
 
-## (#1) Inclusive and community-driven
+## (#1) GUI
 
-- [x] Maintainers from 4 countries, 4 companies
-- [x] Enhancements approved by a community-driven process
-- [ ] Installation documentation in 5+ written languages
+- [ ] Be able to start, stop, pause, and delete clusters via a GUI
```
Comment (afbjorklund) [inline site/content/en/docs/contrib/roadmap.en.md:18]: + [] Application avaiable for all supported platforms: Linux, macOS, Windows

File: site/content/en/docs/contrib/roadmap.en.md (hunk 1 above)
Comment (csantanapr) [inline site/content/en/docs/contrib/roadmap.en.md:30]: Typo: Dependency

**Pull request merged**